<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACabSettingsClass</class>
 <widget class="QDialog" name="ACabSettingsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cabinet Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="upButtonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>201</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cImportSync">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>156</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Import Synchronization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>158</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Export Synchronization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto Compile</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cExportSync">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>290</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cAutoCompile">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>330</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cStorageScope">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>156</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Storage Scope</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayout">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>99</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <widget class="QLineEdit" name="cSourceLocation"/>
    </item>
    <item>
     <widget class="QPushButton" name="cBrowseSource">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayout_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <widget class="QLineEdit" name="cDatabaseLocation"/>
    </item>
    <item>
     <widget class="QPushButton" name="cBrowseDatabase">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>114</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>133</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Database Location</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>133</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Database Name</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayout_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <item>
     <widget class="QLineEdit" name="cDatabaseName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>330</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Switch</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cSearchSwitch">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Lambda Dependencies</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cDependencies">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>391</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cDatabaseName</tabstop>
  <tabstop>cDatabaseLocation</tabstop>
  <tabstop>cBrowseDatabase</tabstop>
  <tabstop>cSourceLocation</tabstop>
  <tabstop>cBrowseSource</tabstop>
  <tabstop>cStorageScope</tabstop>
  <tabstop>cImportSync</tabstop>
  <tabstop>cExportSync</tabstop>
  <tabstop>cAutoCompile</tabstop>
  <tabstop>upButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
